### DISCUSSION TITLE
Conference Presentation Materials

### DISCUSSION THREAD

**Discussion by snarkipus** [eclipse-langium/langium#652] [Q&A] [answered]
Hey Langium Team!

I'm slowly getting to the point where I understand enough about this framework and the underlying concepts to be able to pitch it to my peers. While it's become a personal interest/hobby, I do have a bonafide production use case as the impetus behind this endeavor.

Would it be possible to post any of the conference material for Langium? For instance, although it's targeted at Theia, Irina's charts from her presentation at  [Cloud IDE Days 2022](https://www.youtube.com/watch?v=b46k4D-CAro) would be perfect as an introduction.

Thanks for all the hard work!

**Comment by spoenemann** [comment, marked as answer]
Hi, here are the slides from my presentation at WeAreDevelopers World Congress in June:

[langium-wearedevelopers-2022.pdf](https://github.com/langium/langium/files/9442264/langium-wearedevelopers-2022.pdf)

Here's the recording of that session:

https://youtu.be/rZ85Ib6bjfE

**Reply by snarkipus** [reply]
Thank you @spoenemann !

You all have done a fantastic job with this tool. Every time I think I've hit a roadblock, it seems I discover some elegant utility or helper function. If someone with zero experience in TypeScript or language development can pick this up (like me), then I'd say you're definitely on the right track!

**Comment by pluralia** [comment]
Hi @snarkipus, thank you for your interest!
Here are the slides: [Langium_for_DSLs_in_the_cloud.pdf](https://github.com/langium/langium/files/9476364/Langium_for_DSLs_in_the_cloud.pdf)
